experiments/Marmot/bbox/tikzlibrarybbox.code.tex

```diff
@@ -0,0 +1,161 @@
+\tikzset{%
```

Comment:
After some refactoring I have merged the `bbox` library into PGF/Ti*k*Z.  Thank you for your contribution.